xarray/core/ops.py

```diff
@@ -121,20 +116,6 @@
     None. Changed in version 0.17.0: if specified on an integer array
     and skipna=True, the result will be a float array."""
 
-_COARSEN_REDUCE_DOCSTRING_TEMPLATE = """\
```

Comment:
Where did that go? Or was this superfluous?

Reply:
It was not used.